day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,32 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(topping)
+    toppings.push(topping)
+  end
+
+  def remove_topping(topping)
+    toppings.delete(topping)
+  end
+
+  def change_protein(protein)
+    @protein = protein
```

Comment:
Good job updating the attribute `@protein` directly with the instance variable. It's funky ruby that you're able to access the `@toppings` array via the `attr_reader` and more conventional to use the instance variable `@toppings` itself when modifying it.